lib/sendgrid/helpers/helpers.rb

```diff
@@ -0,0 +1,24 @@
+module SendGrid
+  module Helpers
+
+    # this method name is not accurate.  This method returns a hash of the non-nil instance variables
+    # TODO: update all uses to use :to_hash
+
+    def to_json
+      return nil if nil?
+
+      ret = {}
+      instance_variables.each do |attribute|
+        val = instance_variable_get(attribute)
```

Comment:
Not sure it's a good idea to reach into all instance variables of all classes. What if, in the future someone adds an instance variable that shouldn't be exposed?   
One alternative could be to add a serializable_params (and then iterate through them instead) and register it in all initializers, ie:  
```
def initialize(enable: nil, email: nil)
  @enable = enable
  @email = email
  serializable_params([:enable, :email])
end
```